pkg/interpreter/executor_test.go

```diff
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package executor
+package interpreter
```

Comment:
We'll make SQLFlow a compiler at the end, so do not name it interpreter here?

Reply:
Back to `executor`, that we don't need to modify so many files.